src/foam/u2/dialog/Popup.js

```diff
@@ -105,26 +98,33 @@ foam.CLASS({
           .addClass(this.myClass('inner'))
           .style({ 'background-color': this.backgroundColor })
           .startContext({ data: this })
-            .start(this.CLOSE_MODAL).show(this.closeable$)
+            .start(this.CLOSE_MODAL, { buttonStyle: 'TERTIARY' }).show(this.closeable$)
               .addClass(this.myClass('X'))
             .end()
           .endContext()
-        .end()
-      .end();
+        .end();
 
       this.content = content;
     },
 
     function open() {
       this.document.body.insertAdjacentHTML('beforeend', this.outerHTML);
       this.load();
+      // Sets focus to first modal element after the close button
```

Comment:
This will have race-conditions if any of the child views have async initE() methods. Can't you just let the children set their own auto-focus? Since Input's have an autofocus property and Elements have a focus() method and 'focused' property?

Reply:
We would just have to make sure to go through it in code review because there is no other way to throw focus to the modal elements